bftengine/src/bcstatetransfer/BCStateTran.cpp

```diff
@@ -917,12 +922,25 @@ void BCStateTran::startCollectingStateInternal() {
   startCollectingStateImpl();
 }
 
-void BCStateTran::startCollectingStateImpl() {
+void BCStateTran::setExternalCollectingState(bool newVal) {
+  if (newVal == externalCollectingState_) {
```

Comment:
The logic changed few times so I removed it to prevent too much noise in log, then forgot to add.
With current logic I won't expect it to happen at all. 
Adding, thanks.